lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -1118,15 +1118,22 @@ swift::findTransitiveUsesForAddress(SILValue projectedAddress,
         isa<SelectEnumAddrInst>(user) || isa<InjectEnumAddrInst>(user) ||
         isa<IsUniqueInst>(user) || isa<ValueMetatypeInst>(user) ||
         isa<DebugValueInst>(user) || isa<EndBorrowInst>(user) ||
-        isa<ExplicitCopyAddrInst>(user)) {
-      leafUse(op);
+        isa<ExplicitCopyAddrInst>(user) || isa<DeallocStackInst>(user) ||
+        isa<InitBlockStorageHeaderInst>(user) ||
+        isa<GetAsyncContinuationAddrInst>(user) ||
+        isa<ExistentialMetatypeInst>(user) ||
+        isa<UncheckedRefCastAddrInst>(user) || isa<KeyPathInst>(user) ||
+        isa<RetainValueAddrInst>(user) || isa<ReleaseValueAddrInst>(user) ||
+        isa<PackElementSetInst>(user) || isa<PackElementGetInst>(user) ||
+        isa<DeinitExistentialAddrInst>(user) || isa<LoadBorrowInst>(user)) {
```

Comment:
You may want to check the opcode list against `swift::visitAccessedAddress`

Reply:
I removed the comment.